techniques/css/C41.html

```diff
@@ -1,5 +1,5 @@
-<!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" lang="en"><head>
-		<title>C41: Creating a strong focus indicator within the component | WAI | W3C</title>
+<!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" lang="ja"><head>
+		<title>C41: コンポーネント内に明るいフォーカスインジケータを作成する | WAI | W3C</title>
```

Comment:
strong を「明るい」と訳していますが、暗い色のフォーカスインジケータであるケースもあるかと思います。「強力なフォーカスインジケータ」でもよいかと思いますが、いかがでしょうか？

Reply:
> とはいえ、strongで真っ先に思いつく「強い」はあまりしっくりこない気がしており。

ですよね。(わかります...)

> コンポーネント内に目立つフォーカスインジケータを作成する

なるほど、よさそうです。

```suggestion
		<title>C41: コンポーネント内に目立つフォーカスインジケータを作成する | WAI | W3C</title>
```